evalai/urls.py

```diff
@@ -49,8 +49,8 @@
     url(r"^api/admin/", admin.site.urls),
     url(
         r"^api/auth/login",
-        obtain_expiring_auth_token,
-        name="obtain_expiring_auth_token",
+        token_obtain_pair,
```

Comment:
Why are we changing this?

Reply:
This is to replace `obtain_expiring_auth_token` with `token_obtain_pair` for login.

Reply:
@ShauryaAg we can't use this. If you try login/logout multiple times the JWT token will keep changing. We are using JWT tokens mainly for API and CLI api calls. The expiring token is used for frontend codebase. For this change you just have to add an API call for `get_auth_token` [API](https://github.com/Cloud-CV/EvalAI/blob/master/apps/accounts/views.py#L43) and use the token returned by this API to show on `Submit` page.

Reply:
Currently, the `authtoken` is set in the local storage of the browser during login. Should I just make changes in the frontend to call that `get_auth_token` method everytime instead of setting the `authtoken` in local storage as before?

Reply:
Yes, or you could make API call once and store the bearer token as a separate variable in local storage.